amcl/src/amcl/pf/pf_pdf.c

```diff
@@ -145,6 +145,22 @@ double pf_ran_gaussian(double sigma)
   return(sigma * x2 * sqrt(-2.0*log(w)/w));
 }
 
+// double pf_flat_gaussian_flat(double sigma)
```

Comment:
Not necessary (yet) -- inducing bimodal dist from odom update is a sufficient hack for now.